### PR TITLE
Feature/#505 added quiz fpga.md

### PR DESCRIPTION
**Pull request by Samridhi-98**
Added pop quiz for **fgpa.md**  

Fixes #505 

#### Changes done:
- [x] Task 1
MCQ in the `fpga.md`

#### Screenshots
cvs

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@Shivansh2407  please review my PR.

File: docs/logic-design/kmaps.md [hunk 1]
```diff
@@ -186,3 +186,6 @@ Both groupings have the same size and are the same dimension. However, upon reac
 This illustrates the idea that this is a greedy algorithm, and does not always return the most simplified SOP expression. In later sections, algorithms illustrating a globally optimal algorithm will be discussed.
 
 {% include kmap.html %}
+
+
+{
```
Comment (manjotsidhu) [inline docs/logic-design/kmaps.md:191]: Kindly remove this

File: docs/seq-lsi/fpga.md [hunk 2]
```diff
@@ -51,3 +51,26 @@ FPGA are so versatile than even full microprocessors can be implemented in them,
...
 	webkitAllowFullScreen mozAllowFullScreen allowFullScreen>
 </iframe>
+
+{:.quiz}
+1. Configurable Logic Blocks are surrounded by a system of programmable interconnects is known as
+   1. fabric
+   * chips
+   * flip-flop
+   * finite 
+2. What is the function of D flip flop in FPGA?
```
Comment (manjotsidhu) [inline docs/seq-lsi/fpga.md:61]: Space before "?" is required
```suggestion
2. What is the function of D flip flop in FPGA ?
```

**Review by manjotsidhu — CHANGES_REQUESTED** [review]

**Comment by Samridhi-98** [comment]
@manjotsidhu  @Shivansh2407  resolved all conflicts.

File: docs/seq-lsi/fpga.md (hunk 2 above)
Comment (manjotsidhu) [reply docs/seq-lsi/fpga.md]: Pending

File: docs/logic-design/kmaps.md (hunk 1 above)
Comment (manjotsidhu) [reply docs/logic-design/kmaps.md]: This is important

**Review by Shivansh2407 — APPROVED** [review]
Great work @Samridhi-98 🎉 Please make sure you remember squashing the commits! Thanks a lot 👍

**Pull request merged**